src/test/app/NFToken_test.cpp

```diff
@@ -2872,15 +2872,32 @@ class NFToken_test : public beast::unit_test::suite
             BEAST_EXPECT(ownerCount(env, minter) == 2);
             BEAST_EXPECT(ownerCount(env, buyer) == 1);
 
-            // issuer cannot broker the offers, because they are not the
-            // Destination.
-            env(token::brokerOffers(
-                    issuer, offerBuyerToMinter, offerMinterToBroker),
-                ter(tecNFTOKEN_BUY_SELL_MISMATCH));
-            env.close();
-            BEAST_EXPECT(ownerCount(env, issuer) == 0);
-            BEAST_EXPECT(ownerCount(env, minter) == 2);
-            BEAST_EXPECT(ownerCount(env, buyer) == 1);
+            // fixNFTokenBrokerAccept: Disabled
+            if (!features[fixNFTokenBrokerAccept])
+            {
+                // issuer cannot broker the offers, because they are not the
+                // Destination.
+                env(token::brokerOffers(
+                        issuer, offerBuyerToMinter, offerMinterToBroker),
+                    ter(tecNFTOKEN_BUY_SELL_MISMATCH));
+                env.close();
+                BEAST_EXPECT(ownerCount(env, issuer) == 0);
+                BEAST_EXPECT(ownerCount(env, minter) == 2);
+                BEAST_EXPECT(ownerCount(env, buyer) == 1);
+            }
+            else
+            // fixNFTokenBrokerAccept: Enabled
+            {
+                // issuer cannot broker the offers, because they are not the
+                // Destination.
+                env(token::brokerOffers(
+                        issuer, offerBuyerToMinter, offerMinterToBroker),
+                    ter(tecNO_PERMISSION));
+                env.close();
+                BEAST_EXPECT(ownerCount(env, issuer) == 0);
+                BEAST_EXPECT(ownerCount(env, minter) == 2);
+                BEAST_EXPECT(ownerCount(env, buyer) == 1);
+            }
```

Comment:
I noticed that the only difference between the contents of the `if` and `else` branches is the `tec` code.  Consider reducing the duplication by replacing the `if/else` with something like this:
```
            {
                // issuer cannot broker the offers, because they are not the
                // Destination.
                TER const expectTer = features[fixNFTokenBrokerAccept]
                    ? tecNO_PERMISSION : tecNFTOKEN_BUY_SELL_MISMATCH;
                env(token::brokerOffers(
                        issuer, offerBuyerToMinter, offerMinterToBroker),
                    ter(expectTer));
                env.close();
                BEAST_EXPECT(ownerCount(env, issuer) == 0);
                BEAST_EXPECT(ownerCount(env, minter) == 2);
                BEAST_EXPECT(ownerCount(env, buyer) == 1);
            }
```